libraries/c_sdk/standard/mqtt/test/unit/iot_tests_mqtt_api.c

```diff
@@ -922,6 +1095,9 @@ TEST( MQTT_Unit_API, DisconnectMallocFail )
         _closeCount = 0;
         _disconnectCallbackCount = 0;
     }
+
+    /* Remove the MQTT Context for the MQTT connection. */
+    _IotMqtt_removeContext( _pMqttConnection );
```

Comment:
Does this need to be explicitly cleaned up. Wouldn't library internally cleaning this up?

Reply:
Addressed